plugins/output/flight/flight.go

```diff
@@ -0,0 +1,156 @@
+package flight
+
+import (
+	"context"
+	_ "embed"
+	"log"
+	"net"
+
+	"github.com/apache/arrow/go/v9/arrow"
+	"github.com/apache/arrow/go/v9/arrow/array"
+	"github.com/apache/arrow/go/v9/arrow/flight"
+	"github.com/apache/arrow/go/v9/arrow/ipc"
+	"github.com/apache/arrow/go/v9/arrow/memory"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/outputs"
+)
+
+// DO NOT REMOVE THE NEXT TWO LINES! This is required to embed the sampleConfig data.
+//
+//go:embed sample.conf
+var sampleConfig string
+
+func (*Flight) SampleConfig() string {
+	return sampleConfig
+}
+
+type Flight struct {
+	Location string `toml:"location"`
+	Port     string `toml:"port"`
+	Table    string `toml:"table"`
+
+	timeStamps []arrow.Timestamp
+	values     []float32
+
+	client flight.FlightServiceClient
+	writer flight.Writer
+	schema arrow.Schema
+	ctx    context.Context
+	pool   memory.GoAllocator
+	desc   *flight.FlightDescriptor
+}
+
+// Write should write immediately to the output, and not buffer writes
+// (Telegraf manages the buffer for you). Returning an error will fail this
+// batch of writes and the entire batch will be retried automatically.
+func (s *Flight) Write(metrics []telegraf.Metric) error {
+
+	for _, m := range metrics {
+
+		timeInt := m.Time().UnixMilli()
+
+		s.timeStamps = append(s.timeStamps, arrow.Timestamp(timeInt))
+
+		for _, field := range m.FieldList() {
+			s.values = append(s.values, float32(field.Value.(float64)))
+		}
+
+	}
+
+	b := array.NewRecordBuilder(&s.pool, &s.schema)
+	defer b.Release()
+
+	b.Field(0).(*array.Int32Builder).AppendValues([]int32{1}, nil)
```

Comment:
Maybe document why a tid is needed and why one is used.

Reply:
I still don't quite grasp why it is needed, I only did this implementation to adhere to the schema. Can you quickly elaborate?

Reply:
`tid` was the [unique auto-incrementing integer](https://github.com/ModelarData/ModelarDB/blob/master/src/main/java/dk/aau/modelardb/core/Partitioner.java#L52) that the legacy JVM-based version of ModelarDB used for uniquely identifying each time series. It is currently being removed from the Rust version so maybe add the comment "Unused and deprecated time series identifier." and then remove both the comment and code when it is no longer needed to match the Schema?

Reply:
Sound good. Fixed.